web-app/app/globals.css

```diff
@@ -2,6 +2,15 @@
 @tailwind components;
 @tailwind utilities;
 
+@layer components {
```

Comment:
Using Tailwind layers, we are able to easily reuse the styles while making use of Tailwind's feature set — and we don't need to import anything!